data/json/recipes/recipe_ammo.json

```diff
@@ -27,6 +27,27 @@
   ]
 },{
   "type" : "recipe",
+  "result": "40mm_toxicgas",
+  "category": "CC_AMMO",
+  "subcategory": "CSC_AMMO_BULLETS",
+  "skill_used": "fabrication",
+  "difficulty": 3,
+  "time": 100,
```

Comment:
Just 100 for repurposing the bomb into a contact 'nade and adding propellant so that it can actually be launched?